src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/CngProperty.cs

```diff
@@ -27,6 +27,17 @@ public CngProperty(string name, byte[]? value, CngPropertyOptions options)
             _value = (value == null) ? null : value.CloneByteArray();
```

Comment:
```suggestion
            _value = value.CloneByteArray();
```

[`CloneByteArray` returns `null` if the array is `null`.](https://github.com/vcsjones/runtime/blob/2e92ea584467c02ff04dce9156bb7ef755c3c3f8/src/libraries/Common/src/System/Security/Cryptography/Helpers.cs#L54-L62) The null check could be removed.